code/modules/xenoarchaeology/xenoartifact_traits.dm

```diff
@@ -0,0 +1,1442 @@
+///Xenoartifact traits, datum-ised
```

Comment:
This file just needs to be split down, having a 1400 line long file makes it a massive pain to find, refactor, remove or rework anything contained inside of it and tanks readability.
Instead, add a folder and include the traits inside of that.